src/pages/OrganizationDetail.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { withOidcSecure } from '@axa-fr/react-oidc';
+import { PanelGrid, PanelGridItem } from '../components/PanelGrid';
+import {
+	useOrganization,
+	useOrganizations,
+	ORGANIZATIONS_DEFAULT_PAGE,
+	ORGANIZATIONS_DEFAULT_COUNT,
+} from '../pdc-api';
+import { OrganizationDetailPanel } from '../components/OrganizationDetailPanel';
+import { OrganizationListGridPanel } from '../components/OrganizationListGridPanel';
+
+const OrganizationListGridPanelLoader = () => {
+	const { organizationId } = useParams();
+
+	const [organizations] = useOrganizations(
+		ORGANIZATIONS_DEFAULT_PAGE,
+		ORGANIZATIONS_DEFAULT_COUNT,
+	);
+
+	if (organizations === null) {
+		return <div>Loading data...</div>;
+	}
+
+	return (
+		<PanelGridItem key="detailPanel">
+			<OrganizationListGridPanel
+				organizations={organizations}
+				activeOrganizationId={organizationId}
+			/>
+		</PanelGridItem>
+	);
+};
+
+const OrganizationDetailPanelLoader = () => {
+	const params = useParams();
+	const organizationId = params.organizationId ?? 'missing';
+	const [organization] = useOrganization(organizationId);
+
+	useEffect(() => {
+		if (organization === null) {
+			document.title = 'Organization Detail - Philanthropy Data Commons';
```

Comment:
```suggestion
			document.title = 'Loading… - Philanthropy Data Commons';
```

(I know, this is a change from the proposal page. That's okay!)